src/IntegralArrays.jl

```diff
@@ -1,5 +1,74 @@
 module IntegralArrays
 
-# package code goes here
+using FileIO # for loading images
+using QuartzImageIO, ImageMagick, ImageSegmentation, ImageFeatures # for reading images
+using Colors # for making images greyscale
+using Images # for channelview; converting images to matrices
+using ImageTransformations # for scaling high-quality images down
+
+#=
+Any one pixel in a given image has the value that is the sum of all of the pixels above it and to the left.
+Original    Integral
++--------   +------------
+| 1 2 3 .   | 1  3  6 .
+| 4 5 6 .   | 5 12 21 .
+| . . . .   | . . . . .
+=#
+
+function getImageMatrix(imageFile::AbstractString)
```

Comment:
We follow the [Julia style guide](https://docs.julialang.org/en/latest/manual/style-guide/#Use-naming-conventions-consistent-with-Julia-base/-1) and will probably transition to the [Blue style guide](https://github.com/invenia/BlueStyle) as well, so please using naming consistent with standard Julia choices.